share/spice/riffsy_emoji_search/riffsy_emoji_search.js

```diff
@@ -0,0 +1,36 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_riffsy_emoji_search = function(api_result){
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.error || !api_result.results) {
```

Comment:
@bryanhart When the query is invalid results can still be an empty array. `![]` will result to false so probably best to check `api_result.results.length` as well.